types/is-secret/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for is-secret 1.2
+// Project: https://github.com/watson/is-secret#readme
+// Definitions by: wrumsby <https://github.com/wrumsby>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace isSecret;
+export = isSecret;
+
+declare namespace isSecret {
+    function key(key: string): boolean;
+
+    function value(value: string): boolean;
+}
```

Comment:
```suggestion
export as namespace isSecret;

export function key(key: string): boolean;

export function value(value: string): boolean;

```
if you accept the export as top level funcs, apply this, otherwise skip